### ISSUE TITLE
New-AppSourceSubmission appears to "timeout" and just continuously reauthenticate

### ISSUE THREAD

**Issue by JackVernonEliteDynamics** [microsoft/navcontainerhelper#2673]
**Describe the issue**
When the Get-AppSourceSubmission function runs, it will print period character as it waits.
After a period of time (30 minutes +) this stops and the script continues reauthenticates with the api.partner.microsoft.com endpoint.

**Scripts used to create container and cause the issue**
```
Get-AppSourceSubmission -authContext $authcontext -productId $productId -includeWorkflowDetails
```

**Full output of scripts**
```
     BcContainerHelperis version 4.0.0-preview690
 BcContainerHelperis is running as administrator
...
```

**Screenshots**
![script](https://user-images.githubusercontent.com/66066971/188509251-293cd2f6-2f4e-4755-abc0-06a17b8093ed.png)

**Additional context**
- does it happen all the time? - Yes
- did it use to work? - No

**Comment by freddydk** [comment]
Yeah, the access token is expiring, which makes the function grab a new one for every call.
It should still work - but the output is cluddered.
Will renew the token and cache it in a different location (and doing it silently)

**Comment by freddydk** [comment]
Shipped in 4.0.3
